client/components/jetpack/add-stored-credit-card/index.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement } from 'react';
+import { useDispatch } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'calypso/components/card-heading';
+import Gridicon from 'calypso/components/gridicon';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function AddStoredCreditCard(): ReactElement {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const navigateToCreateMethod = () => {
+		dispatch(
+			recordTracksEvent( 'calypso_partner_portal_license_list_empty_issue_license_click' )
+		);
+
+		page( '/partner-portal/payment-method/add' );
+	};
+
+	return (
+		<div
+			className="add-stored-credit-card"
+			onClick={ navigateToCreateMethod }
+			onKeyDown={ navigateToCreateMethod }
+			role="button"
+			tabIndex={ 0 }
+		>
+			<div className="add-stored-credit-card__content">
+				<CardHeading className="add-stored-credit-card__text" size={ 18 }>
+					{ translate( 'Add new credit card' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Add a new credit card' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-partner-portal-payment-method-list)